src/game.cpp

```diff
@@ -12662,7 +12668,7 @@ void game::plswim(int x, int y)
     if (u.is_underwater()) {
         drenchFlags |= mfb(bp_head) | mfb(bp_eyes) | mfb(bp_mouth) | mfb(bp_hand_l) | mfb(bp_hand_r);
     }
-    u.drench(100, drenchFlags);
+    u.drench( 100, drenchFlags, true );
```

Comment:
(Possible pre-existing bug) This is not called when the character just stays in the water (doesn't move, but is technically swimming). That makes testing this a bit complicated - if the character moves into deep water and just stays there and waits, the wetness will eventually reach 0.